packages/README.md

```diff
@@ -11,10 +15,10 @@ in compliance with [Flutter repo style guide]( https://github.com/flutter/flutte
 2. Private field
 3. Public setter (when needed)
 
-## Naming for methods and functions
+## Naming for function variables
 
 In compliance with [Flutter repo style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#naming-rules-for-typedefs-and-function-variables) we name:
 
-1. Typedefs: `FooCallback`
-2. Arguments and properties: `onFoo`
-3. Methods and functions: `handleFoo`
+1. Typedefs for functions: `FooCallback`
+2. Arguments and properties for functions: `onFoo`
+3. Methods and functions for event handling: `handleFoo`
```

Comment:
Even with these edits, this short form doesn't really reflect the content of the original link.
I think we should just include the link. It will avoid duplication as well.
```suggestion
1. Use Typedefs to define callbacks: `FooCallback`
2. For callback argument/property use: `onFoo`
3. For a method that is passed as a callback: `handleFoo`
```